tests/examples/mix.py

```diff
@@ -1,8 +1,10 @@
 from pyspark.sql import functions as F
 from tests.context import spark
 
+# convert into table
+spark.createDataFrame([], schema="struct<dni:int, name:string, age:int, weight:float, city:string>").write.saveAsTable("people")
 
-people = spark.createDataFrame([], schema="struct<dni:int, name:string, age:int, weight:float, city:string>")
+people = spark.table("people")
```

Comment:
Remember adding the corresponding unit test

Reply:
done